2020-10-15 12:22:18.752142: 
Update ERROR: Delete from items where item_id=152no results to fetch

2020-10-15 12:22:18.752412: 
Traceback (most recent call last):
  File "/home/vinay.rastogi/Documents/ThinkBridge/shopbridge-backend/DatabaseUtil.py", line 73, in delete_item
    results = self.mycursor.fetchall()
psycopg2.ProgrammingError: no results to fetch


2020-10-15 12:22:18.781469: 
Update ERROR: Delete from items where item_id=152no results to fetch

2020-10-15 12:22:18.781670: 
Traceback (most recent call last):
  File "/home/vinay.rastogi/Documents/ThinkBridge/shopbridge-backend/DatabaseUtil.py", line 73, in delete_item
    results = self.mycursor.fetchall()
psycopg2.ProgrammingError: no results to fetch


2020-10-15 12:22:51.646415: 
Update ERROR: Delete from items where item_id=153no results to fetch

2020-10-15 12:22:51.646567: 
Traceback (most recent call last):
  File "/home/vinay.rastogi/Documents/ThinkBridge/shopbridge-backend/DatabaseUtil.py", line 73, in delete_item
    results = self.mycursor.fetchall()
psycopg2.ProgrammingError: no results to fetch


2020-10-15 12:22:51.678458: 
Update ERROR: Delete from items where item_id=153no results to fetch

2020-10-15 12:22:51.678619: 
Traceback (most recent call last):
  File "/home/vinay.rastogi/Documents/ThinkBridge/shopbridge-backend/DatabaseUtil.py", line 73, in delete_item
    results = self.mycursor.fetchall()
psycopg2.ProgrammingError: no results to fetch


2020-10-15 12:23:32.657515: 
Update ERROR: Delete from items where item_id=154no results to fetch

2020-10-15 12:23:32.657788: 
Traceback (most recent call last):
  File "/home/vinay.rastogi/Documents/ThinkBridge/shopbridge-backend/DatabaseUtil.py", line 73, in delete_item
    results = self.mycursor.fetchall()
psycopg2.ProgrammingError: no results to fetch


2020-10-15 12:23:32.688121: 
Update ERROR: Delete from items where item_id=154no results to fetch

2020-10-15 12:23:32.688270: 
Traceback (most recent call last):
  File "/home/vinay.rastogi/Documents/ThinkBridge/shopbridge-backend/DatabaseUtil.py", line 73, in delete_item
    results = self.mycursor.fetchall()
psycopg2.ProgrammingError: no results to fetch


2020-10-15 12:24:09.292338: 
Update ERROR: Delete from items where item_id=155no results to fetch

2020-10-15 12:24:09.292494: 
Traceback (most recent call last):
  File "/home/vinay.rastogi/Documents/ThinkBridge/shopbridge-backend/DatabaseUtil.py", line 73, in delete_item
    results = self.mycursor.fetchall()
psycopg2.ProgrammingError: no results to fetch


2020-10-15 12:24:09.325165: 
Update ERROR: Delete from items where item_id=155no results to fetch

2020-10-15 12:24:09.325349: 
Traceback (most recent call last):
  File "/home/vinay.rastogi/Documents/ThinkBridge/shopbridge-backend/DatabaseUtil.py", line 73, in delete_item
    results = self.mycursor.fetchall()
psycopg2.ProgrammingError: no results to fetch


2020-10-15 12:25:38.176869: 
Update ERROR: Delete from items where item_id=155no results to fetch

2020-10-15 12:25:38.177059: 
Traceback (most recent call last):
  File "/home/vinay.rastogi/Documents/ThinkBridge/shopbridge-backend/DatabaseUtil.py", line 73, in delete_item
    results = self.mycursor.fetchall()
psycopg2.ProgrammingError: no results to fetch


2020-10-15 12:26:21.993428: 
Update ERROR: Delete from items where item_id=155no results to fetch

2020-10-15 12:26:21.994122: 
Traceback (most recent call last):
  File "/home/vinay.rastogi/Documents/ThinkBridge/shopbridge-backend/DatabaseUtil.py", line 73, in delete_item
    results = self.mycursor.fetchall()
psycopg2.ProgrammingError: no results to fetch


2020-10-15 12:26:43.066558: 
Update ERROR: Delete from items where item_id=123no results to fetch

2020-10-15 12:26:43.066747: 
Traceback (most recent call last):
  File "/home/vinay.rastogi/Documents/ThinkBridge/shopbridge-backend/DatabaseUtil.py", line 73, in delete_item
    results = self.mycursor.fetchall()
psycopg2.ProgrammingError: no results to fetch


